### PR TITLE
Adding directions about mounting in `kube-controller-manager`

### PR DESCRIPTION
**Pull request by tianhuil**
Based on [this issue](https://github.com/kubernetes-incubator/external-storage/issues/529).  @klausenbusk, take a look.

### CODE REVIEW

File: digitalocean/README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
 3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`
+   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`.
+   - Please ensure the flexvolume plugin directory path is accessible in `kube-controller-manager` by adding a hostPath mount to the manifest file on the master node (`/etc/kubernetes/manifests/kube-controller-manager.yaml`), using [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml) as an example.
```
Comment (klausenbusk) [inline digitalocean/README.md]: I'm wondering if we could rephrase the sentence in a better way. Something like: (maybe, I'm not good at english at al)

> **Note:** Please ensure that `kube-controller-manager` has access to the flexvolume plugin directory. If running `kube-controller-manager` as a pod, please take a look at [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml), for how this can be done.

File: digitalocean/README.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
 3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`
+   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`.
```
Comment (klausenbusk) [inline digitalocean/README.md]: I'm not sure if we should end the point/sentences with period, most of the points in the README doesn't.

File: digitalocean/README.md (hunk 1 above)
Comment (klausenbusk) [reply digitalocean/README.md]: We should probably also document the required `bootkube` tweak I use, as documented in https://github.com/kubernetes-incubator/external-storage/issues/531. Then we could close that issue after merging.

**Comment by tianhuil** [comment]
Done!

File: digitalocean/README.md [hunk 3]
```diff
@@ -8,8 +8,7 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    `base64` the token and insert it into [manifests/digitalocean-secret.yaml](manifests/digitalocean-secret.yaml).  
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
-3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`.
```
Comment (klausenbusk) [inline digitalocean/README.md]: Why was this removed?

File: digitalocean/README.md [hunk 4]
```diff
@@ -8,8 +8,8 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    `base64` the token and insert it into [manifests/digitalocean-secret.yaml](manifests/digitalocean-secret.yaml).  
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
-3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`
+3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml).
+   - *Note:* Please ensure that kube-controller-manager has access to the flexvolume plugin directory. If running `kube-controller-manager` as a pod, please take a look at [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml), for how.
```
Comment (klausenbusk) [inline digitalocean/README.md]: Forgot to highlight the first `kube-controller-manager`

**Comment by wongma7** [comment]
need cncf, minor changes, & there is now a conflict (sorry). Closing for now
